test/common_utils.py

```diff
@@ -132,25 +124,16 @@ def remove_prefix_suffix(text, prefix, suffix):
 
         return expected_file
 
-    def assertExpected(self, output, subname=None, prec=None, strip_suffix=None):
+    def assertExpected(self, output, name, prec=None):
```

Comment:
I had to tweak this a little because  in the previous version, the check name was based on `self.id()`, which changes now that the tests are parametrized with pytest.

In fact I simplified it a bit because all what `assertExpected` needs to know is the name of the model, so I removed `strip_suffix` (which we don't need anymore) and also `subname` which was actually never used.

Reply:
As a general note I would like us to re-use as much as possible the testing utils from PyTorch, if it makes sense. This was originally taken from PyTorch tests, and now it is exposed via `torch.testing` I believe